database.sql

```diff
@@ -855,6 +855,7 @@ CREATE TABLE IF NOT EXISTS `queue` (
 	`id` int NOT NULL auto_increment COMMENT '',
 	`cid` int NOT NULL DEFAULT 0 COMMENT '',
 	`network` varchar(32) NOT NULL DEFAULT '' COMMENT '',
+	`guid` varchar(255) NOT NULL DEFAULT '' COMMENT '',
```

Comment:
Could you add it to the DB structure docs as well?

Reply:
Eventually the documentation will be created from the database structure, that's why there is now this comment field. So I would postpone the manual editing until that time. (That should be in this release as well).

Reply:
See issue https://github.com/friendica/friendica/issues/4419 and https://github.com/friendica/friendica/issues/4420

Reply:
But it would be nice if you could insert something in the new comment field and don't leave it empty ;-)

Reply:
Documentation done. :-P

Reply:
I can’t see the field comment, did you forget to commit/push it?

Reply:
Note to myself: Commit before push, not the reverse ...